### PR TITLE
Unused "next" trace

### PR DESCRIPTION
**Pull request by ygaller**

### CODE REVIEW

File: src/main/java/io/opentracing/contrib/jdbi/OpenTracingCollector.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public OpenTracingCollector(Tracer tracer) {
      */
     @SuppressWarnings("unused")
```
Comment (malafeev) [inline src/main/java/io/opentracing/contrib/jdbi/OpenTracingCollector.java]: could you also remove `@SuppressWarnings("unused")`?

**Comment by ygaller** [comment]
Issue #12

**Comment by malafeev** [comment]
test failed

File: src/test/java/io/opentracing/contrib/jdbi/OpenTracingCollectorTest.java [hunk 2]
```diff
@@ -76,6 +76,38 @@ public void testParentage() {
...
+            }
+        };
+
+        TestTimingCollector subject = new TestTimingCollector();
+
+        dbi.setTimingCollector(new OpenTracingCollector(tracer, subject));
+
+        // The actual JDBI code:
+        {
+            Handle handle = dbi.open();
```
Comment (malafeev) [inline src/test/java/io/opentracing/contrib/jdbi/OpenTracingCollectorTest.java]: please use `try (Handle handle = dbi.open()) {...`

**Pull request merged**